solidity/contracts/ConnextVestingWallet.sol

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.20;
+
+import {VestingWalletWithCliff} from './VestingWalletWithCliff.sol';
+import {IERC20} from '@openzeppelin/contracts/token/ERC20/IERC20.sol';
+
+contract ConnextVestingWallet is VestingWalletWithCliff {
+  address constant BENEFICIARY_ADDRESS = address(420);
+  address constant CONNEXT_TOKEN_ADDRESS = address(420);
+
+  uint64 constant ONE_YEAR = 365 days;
+  uint64 constant AUG_01_2023 = 12_345_678;
+  uint64 constant VESTING_OFFSET = ONE_YEAR - ONE_YEAR / 12;
```

Comment:
its like one month? why?

Reply:
is 11 months, is the offset we do to the VestingContract to replicate the intended behaviour.
if we set the starting date to `AUG_01_2023` (this is a random date btw), then the cliff date to `2024` (1 yr after), when the cliff gives us tokens, will give us for 1yr of a total of 2ys (50%), when it should give us 1/13 at that date